deepspeed/module_inject/auto_tp.py

```diff
@@ -476,7 +481,9 @@ def _replace_module(self, r_module, prev_name='', prev_class_name=''):
 
     def get_model_num_kv_heads(self, config):
         num_kv_heads = None
-        kv_head_names = ['num_kv_heads', 'num_key_value_heads', 'num_attention_heads', 'n_heads']
+        kv_head_names = [
+            'multi_query_group_num', 'num_kv_heads', 'num_key_value_heads', 'num_attention_heads', 'n_heads'
```

Comment:
@Yejing-Lai as this list gets longer, it would help if newly added entries can have comments for what model this new entry serves.  In case one day this piece information is needed.